src/__tests__/DropdownMenu.tsx

```diff
@@ -0,0 +1,119 @@
+import {cleanup, render as HTMLRender, act, fireEvent} from '@testing-library/react'
+import 'babel-polyfill'
+import {axe, toHaveNoViolations} from 'jest-axe'
+import React from 'react'
+import theme from '../theme'
+import {DropdownMenu, DropdownButton} from '../DropdownMenu'
+import {COMMON} from '../constants'
+import {behavesAsComponent, checkExports} from '../utils/testing'
+import {ThemeProvider} from 'styled-components'
+import {BaseStyles} from '..'
```

Comment:
We should use our custom ThemeProvider instead of the styled-components one.
```suggestion
import {BaseStyles, ThemeProvider} from '..'
```

Reply:
Good catch.  Fixed in https://github.com/primer/components/pull/1135/commits/33863d8276f47d2addfd2aef0d2554569edb4a38.  Found one other use as well